test/routes/api/tasks/tasks.test.ts

```diff
@@ -432,7 +432,7 @@ describe('Tasks api (logged user only)', () => {
     })
   })
 
-  describe('Task image upload and retrieval', () => {
+  describe('Task image upload and retrieval and delete', () => {
```

Comment:
You can keep the title I think or create a more generic title.
Adding `and` is the sign that we are doing too much things in the same place.
Maybe we should create 3 (child?) `describe` that share some utilities.

Reply:
Actually, all tests in the task image bloc share the same state, because in retrival & deletion tests, we act on the image uploaded in the first upload test ! what do you think about just making a generic title and splitting test in describe child ?

Reply:
> all tests in the task image bloc share the same state
I know, if achievable with child describe, it's fine.